src/ppmseq/ugbio_ppmseq/ppmSeq_utils.py

```diff
@@ -2024,17 +2023,17 @@ def ppmseq_qc_analysis(  # noqa: C901, PLR0912, PLR0913, PLR0915 #TODO: refactor
             parameters["strand_ratio_category_concordance_png"] = output_strand_ratio_category_concordance_plot
         if sorter_stats_json:
             parameters["output_read_length_histogram_plot"] = output_read_length_histogram_plot
-        # inject parameters and run notebook
-        papermill_params = f"{' '.join([f'-p {k} {v}' for k, v in parameters.items()])}"
-        papermill_cmd = f"papermill {template_notebook} {output_report_ipynb} {papermill_params} -k python3"
-        subprocess.check_call(papermill_cmd.split())
-        # convert to html
-        subprocess.check_call(f"jupyter nbconvert {output_report_ipynb} --to html --no-input".split())
-        # edit html for readability
-        modify_jupyter_notebook_html(output_report_html)
-
-    # remove temporary png and ipynb files
-    if not keep_temp_visualization_files:
-        for visualization_file in output_visualization_files:
-            if os.path.isfile(visualization_file):
-                os.remove(visualization_file)
+
+        # remove temporary png and ipynb files
```

Comment:
you don't actually remove them here, I think the comment should be updated to something like:
`# collect temporary png and ipynb files`